Week3/Day2/src/pipeline/database_connection.py

```diff
@@ -0,0 +1,23 @@
+import psycopg2
+
+def databaseConnect(database_name):
+    try:
+        return psycopg2.connect(
+            user='postgres',
+            password='admin',
+            host='localhost',
+            port='5432',
+            database=database_name
```

Comment:
find a way to not include these variables in git.
You can use google.

Don't use the database connection like this in the final project/assignment.

Reply:
ok dai I will use placeholder and manage by environment variables in final.